Purchases/Purchasing/StoreKitAbstractions/StoreProduct.swift

```diff
@@ -63,6 +71,24 @@ public typealias SK2Product = StoreKit.Product
     @objc public var discounts: [PromotionalOffer] { fatalError() }
 }
 
+public extension StoreProduct {
+
+    /// Calculates the price of this subscription product per month.
+    /// - Returns: `nil` if the product is not a subscription.
+    @available(iOS 11.2, macOS 10.13.2, tvOS 11.2, watchOS 6.2, *)
+    @objc var pricePerMonth: NSDecimalNumber? {
```

Comment:
Note: this has to be an `NSDecimalNumber?` instead of `Decimal?` because the latter isn't `@objc` compatible (even though it could just bridge to the same thing..

Reply:
Did you file a 📡 ?

Reply:
I don't think it's really a bug, just not supported.
It's the same reason you can't write `@objc func f() -> Int? {` and you need `NSNumber?`.

Reply:
ohh, so `Decimal` would work, but `Decimal?` wouldn't? That's... sad. 
Maybe we should update `price` too then, for consistency at least? It'd be weird if you had `price` Decimal and `pricePerMonth` as `NSDecimalNumber`. 

Otherwise we could go a bit extra and provide two properties, one that's `NSDecimalNumber`, marked `@objc` and `@unavailable(swift)` so it's only usable in objc, and then one that's `Decimal` for Swift only

Reply:
> Maybe we should update price too then, for consistency at least? It'd be weird if you had price be Decimal and pricePerMonth be NSDecimalNumber.

Thanks to the generally good bridging behavior I'd say we don't need to. In Objective-C all of these become `NSDecimalNumber`, and in Swift they bridge transparently just like `NSString` and `String` do.